R/Driver.R

```diff
@@ -78,6 +78,7 @@ setMethod(
 #' \url{https://www.connectionstrings.com} is also a useful resource of example
 #' connection strings for a variety of databases.
 #' @aliases dbConnect
+#' @importFrom rlang is_call
```

Comment:
It's not super important here, but in tidyverse packages we general just do `@import rlang`.

Reply:
I'm curious to know the reason behind this preference. Could not find a reference to it in the tidyverse style guide (https://github.com/tidyverse/style). Have a great day.

Reply:
Got it. Thanks for taking the time to write down an explanation. Let me switch it.